mmcv/ops/iou3d.py

```diff
@@ -19,21 +109,28 @@ def boxes_iou_bev(boxes_a, boxes_b):
     Returns:
         torch.Tensor: IoU result with shape (M, N).
     """
+    warnings.warn(
+        '`iou3d.boxes_iou_bev` is deprecated and will be removed in'
+        ' the future. Please, use `box_iou_rotated.box_iou_rotated`.',
+        DeprecationWarning)
     ans_iou = boxes_a.new_zeros(
         torch.Size((boxes_a.shape[0], boxes_b.shape[0])))
+    boxes_a = _xyxyr2xyzwlhr(boxes_a)
```

Comment:
It might be better to directly use box_iou_rotated.box_iou_rotated here, which makes boxes_iou_bev as a wrapper of box_iou_rotated.box_iou_rotated. In this way we do not need `_xyxyr2xyzwlhr` and the code is more straightforward, which tells readers that they are indeed the same as box_iou_rotated.box_iou_rotated